src/components/ProviderInitializer/helpers/handleGuardianWarning.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { faWarning } from '@fortawesome/free-solid-svg-icons';
+
+import { AccountType } from 'types';
+import { GuardianWarningToast } from 'UI';
```

Comment:
use absolute paths for imports for three shaking

Reply:
Updated. 👍